hivemind/p2p/p2p_daemon.py

```diff
@@ -396,12 +402,18 @@ async def add_protobuf_handler(
         input_protobuf_type: type,
         *,
         stream_input: bool = False,
+        stream_output: bool = False,
     ) -> None:
         """
         :param stream_input: If True, assume ``handler`` to take ``TInputStream``
                              (not just ``TInputProtobuf``) as input.
+        :param stream_output: If True, assume ``handler`` to return ``TOutputStream``
         """
 
+        if not (stream_input or stream_output):
```

Comment:
nit: This may be more clear:

```suggestion
        if not stream_input and not stream_output:
```